great_expectations/core/yaml_handler.py

```diff
@@ -0,0 +1,69 @@
+import io
```

Comment:
Love this. I saw in our new style guide an example of a module docstring with "Typical Usage Example:" https://google.github.io/styleguide/pyguide.html#382-modules - does that seem appropriate here?

Reply:
This is a great suggestion :) Thank you @anthonyburdi